Resources/Prototypes/Flavors/flavors.yml

```diff
@@ -194,6 +194,11 @@
   flavorType: Base
   description: flavor-base-chewy
 
+- type: flavor
```

Comment:
same deal regarding the file structure in this and every other file. if you have questions shoot them @ me or post them in dev chat